presto-kudu/src/main/java/com/facebook/presto/kudu/KuduMetadata.java

```diff
@@ -357,13 +358,13 @@ public ColumnHandle getDeleteRowIdColumnHandle(ConnectorSession session, Connect
     }
 
     @Override
-    public ConnectorTableHandle beginDelete(ConnectorSession session, ConnectorTableHandle tableHandle)
+    public ConnectorDeleteTableHandle beginDelete(ConnectorSession session, ConnectorTableHandle tableHandle)
```

Comment:
you need to add implements ConnectorDeleteTableHandle for KuduTableHandle. Otherwise the cast would fail.